modules/con_adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts_{context}"]
+= Adding OpenShift Routes as a traffic manager for Argo Rollouts
+
+With traffic management, a user can modify the amount of traffic to direct it to various applications in a cluster environment. Users can choose to reduce or increase the traffic based on their requirements. 
+
+By enabling routes as traffic managers, a user using OpenShift in their environment ensures that new changes roll out adequately to all the cluster applications, enabling progressive delivery.
+
+You can use Openshift Route as a plugin and automatically install the plugin with the `RolloutManager` custom resource (CR).
+
+[discrete]
+== About OpenShift routes 
+
+An link:https://docs.openshift.com/container-platform/4.14/networking/routes/route-configuration.html[OpenShift Container Platform route] exposes a link:https://docs.openshift.com/container-platform/3.11/architecture/core_concepts/pods_and_services.html#services[service] at a host name, such as www.example.com, so that external clients can reach it by name.
```

Comment:
hello @ishitasequeira - As pointed out correctly by my peer Sri, the link provided for the Services is pointing to a previous version of the documentation in the Word document that you shared. Can we use a newer link from the latest version of the documentation?

The problem with using the latest version is that the content from the older version has been removed and there are no traces of it in the new version. I could only find a one line description of Services in the following link:

https://docs.openshift.com/container-platform/4.13/architecture/index.html

Do you think that we can direct the customers to referring this description or quash the link for the text "Services" completely?

Reply:
As discussed during the call, the link for Services can be quashed.